src/CoreBundle/Bridge/Chameleon/Migration/Script/update-1549624862.inc.php

```diff
@@ -0,0 +1,106 @@
+<h1>Build #1549624862</h1>
+<h2>Date: 2019-02-08</h2>
+<div class="changelog">
+    - Mark old main menu rendering fields as deprecated.
+</div>
+<?php
+
+// cms_tbl_conf
+
+$fieldId = TCMSLogChange::GetTableFieldId(TCMSLogChange::GetTableId('cms_tbl_conf'), 'cms_content_box_id');
+
+$data = TCMSLogChange::createMigrationQueryData('cms_field_conf', 'en')
+  ->setFields([
+      '049_helptext' => '@deprecated since 6.3.0 - use sidebar main menu items instead',
+  ])
+  ->setWhereEquals([
+      'id' => $fieldId,
+  ])
+;
+TCMSLogChange::update(__LINE__, $data);
+
+$data = TCMSLogChange::createMigrationQueryData('cms_field_conf', 'de')
+    ->setFields([
+        '049_helptext' => '@deprecated since 6.3.0 - use sidebar main menu items instead',
+    ])
+    ->setWhereEquals([
+        'id' => $fieldId,
+    ])
+;
+TCMSLogChange::update(__LINE__, $data);
+
+$fieldId = TCMSLogChange::GetTableFieldId(TCMSLogChange::GetTableId('cms_tbl_conf'), 'icon_font_css_class');
+
+$data = TCMSLogChange::createMigrationQueryData('cms_field_conf', 'en')
+  ->setFields([
+      '049_helptext' => 'The field is used to display a font icon to the menu item of the table. Fill in the class name here, for example for Font Awesome: fas fa-check
```

Comment:
Did we already have icon fonts in 6.2?

Reply:
We introduced it in 6.3.0 in a prior PR. It's a bit unusual to mark a new field as deprecated, but this is intended. The field is required to display the classic main menu from 6.3.0 on, but the classic menu itself is deprecated. So this field and the classic menu share a common lifecycle (the unusual thing here is that a deprecated feature is improved).